Resources/Prototypes/Loadouts/uniform.yml

```diff
@@ -436,6 +436,217 @@
       jobs:
         - Passenger
 
+- type: loadout
```

Comment:
So I think we're actually going to need to start addressing a growing problem with loadouts, and that's an issue primarily with certain departments(Command and Security) being easily rendered unrecognizable. But luckily, we have options now where it's possible to make it so that items can "Require one of a specified list of departments", or "Require you not be in a specific department", and for the majority of stuff on at least the Uniforms list it's going to end up being "Not Security or Command", largely for the sake of gameplay issues. 

So long as a clothing item doesn't severely disrupt your ability to tell that someone is a member of Security or Command, it's alright to keep it available to Sec. That unfortunately means most jumpsuits and things like Kimonos or schoolgirl outfits.

Reply:
Resolved with https://github.com/Simple-Station/Einstein-Engines/pull/616/commits/f8ac0c4d3236a2206a46760dc24a735252f8e8ae

These uniforms are visibly red and are still able to be selected by Security:
![image](https://github.com/user-attachments/assets/f73cda1d-271a-4c75-9fa5-edc687f05c5a)